### PR TITLE
Printing the duration of songs from a playlist in HH:MM:SS format

### LINKED ISSUE #1: Task description
Playlista

### PR DESCRIPTION
**Pull request by wojciechkostecki**
Closes #1

### CODE REVIEW

File: src/Playlist.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    public int playlistLength(Song... songs ){
+        int playingTime = 0;
+        for (Song song : songs) {
+            playingTime += song.getDuration();
+        }
+        return playingTime;
+    }
+
+    public void printPlaylistDuration(int durationSong){
```
Comment (rbobko93) [inline src/Playlist.java:15]: Why are you complicating things so much? We just need to turn seconds into actual time. Did you google how to do it? Hint -> https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html

**Review by rbobko93 — CHANGES_REQUESTED** [review]

File: src/Playlist.java (hunk 1 above)
Comment (wojciechkostecki) [reply src/Playlist.java:15]: I did not find a method here that would convert seconds to HH: MM: SS format.

Comment (wojciechkostecki) [reply src/Playlist.java:15]: I'd have to create an additional method that would replace any part of the Duration object with a String.

Comment (rbobko93) [reply src/Playlist.java:15]: There is a way to do it using this class. Look closer.

**Pull request merged**